share/smack/lib/smack.c

```diff
@@ -1006,6 +1011,24 @@ void __SMACK_decls() {
 #undef D
 }
 
+// The size parameter represents number of bits that are being accessed.
+// Whereas when memory is allocated, malloc function in boogie is passed 
```

Comment:
Update this comment based on your latest implementation. I guess you can completely remove it.